Automattic/WP-Job-Manager
# Style settings page header

Fixes 163-gh-Automattic/wpjm-addons

Also moved the Save button to the header, because it can be hard to find/remember sometimes when the page is long.
+ Added the styled header to the Marketplace page too.

### Changes Proposed in this Pull Request

* Add a header with logo to the Settings and Marketplace changes
* Tweak some colors and spacing, set max width
* Move the Save Changes button to the header, make it change color when there are changes to save
* Fix some scroll jumping
* Fix search in the Marketplace page going to the old `..-addons` URL
* Update the outdated lost license link in the License page

### Testing Instructions

* Open the Job Manager → Settings page, click around, change some settings, make sure everything works right
* Also check out the Job Manager → Marketplace page


### Release Notes

* Update design for settings and marketplace pages


### Screenshot / Video

![image](https://github.com/Automattic/WP-Job-Manager/assets/176949/4450807b-e21b-4a8f-98aa-75a9ace8b0a7)

![image](https://github.com/Automattic/WP-Job-Manager/assets/176949/f9831193-ec39-459d-80b6-b1d5c9e25c8b)




----

| Plugin build for 0b81e90db5918d4c4e3e4f0f96434fd3aa73fa73 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/11/wp-job-manager-zip-2641-0b81e90d.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/11/2641-0b81e90d)             |
